ldm/invoke/model_cache.py

```diff
@@ -215,6 +217,8 @@ def _load_model(self, model_name:str):
         tic = time.time()
 
         # this does the work
+        if not os.path.isabs(config):
+            config = os.path.join(Globals.root,config)
```

Comment:
Does this not need `normpath`?

Reply:
normpath() only pretties up the path, doesn't change what it points to. I called it in a few places where it would be displayed to the user.